runtime/runtime/src/state_viewer/mod.rs

```diff
@@ -143,8 +143,9 @@ impl TrieViewer {
         let query = trie_key_parsers::get_raw_prefix_for_contract_data(account_id, prefix);
         let acc_sep_len = query.len() - prefix.len();
         let mut iter = state_update.trie().iter()?;
+        iter.remember_visited_nodes(true);
         iter.seek(&query)?;
-        for item in iter {
+        for item in &mut iter {
```

Comment:
```
error[E0277]: `&TrieIterator<'_>` is not an iterator
   --> runtime/runtime/src/state_viewer/mod.rs:148:21
    |
148 |         for item in &iter {
    |                     -^^^^
    |                     |
    |                     `&TrieIterator<'_>` is not an iterator
    |                     help: consider removing the leading `&`-reference
    |
    = help: the trait `Iterator` is not implemented for `&TrieIterator<'_>`
    = note: `Iterator` is implemented for `&mut near_store::TrieIterator<'_>`, but not for `&near_store::TrieIterator<'_>`
    = note: required because of the requirements on the impl of `IntoIterator` for `&TrieIterator<'_>`
```